api/src/routes/documents/api-tests/get-documents/19_auth_forgot_password.hurl

```diff
@@ -0,0 +1,58 @@
+# Forgot password with valid email
+POST http://localhost:3000/api/auth/forgot-password
+Content-Type: application/json
```

Comment:
```suggestion
```

The content type is not required, from hurl docs: https://hurl.dev/docs/request.html#json-body